lib/doorkeeper/orm/mongo_mapper.rb

```diff
@@ -6,6 +6,14 @@ def self.initialize_models!
         require 'doorkeeper/orm/mongo_mapper/access_token'
         require 'doorkeeper/orm/mongo_mapper/application'
       end
+
+      def self.initialize_application_owner!
+        require 'doorkeeper/models/concerns/ownership'
+
+        Doorkeeper::Application.send :include, Doorkeeper::Models::Ownership
+      end
+
+      def self.check_requirements(config); end
```

Comment:
Unused method argument - `config`. If it's necessary, use `_` or `_config` as an argument name to indicate that it won't be used. You can also write as `check_requirements(*)` if you want the method to accept any arguments but don't care about them.